DiagnosticsExtension/Models/ConnectionStringValidator/Constants.cs

```diff
@@ -14,38 +14,51 @@ namespace DiagnosticsExtension.Models.ConnectionStringValidator
+        public const string EventHubAccessRestrictedDetails = "This can be due to firewall rules on the resource.  Please check if you have configured firewall rules or a private endpoint and that they correctly allow access from the Function App.  See <a href= 'https://docs.microsoft.com/en-us/azure/event-hubs/network-security' target='_blank'>Event Hubs network security</a> for additional details.";
+        public const string DnsLookupFailed = "The service resource specified in the connection string was not found. Please check the value of the setting.";
+        public const string FQNamespaceResourceNotFound = "The resource specified in the app setting '{0}" + FullyQualifiedNamespace + "' was not found. Please check the value of the setting.";
+        public const string StorageAccountResourceNotFound = "The resource specified in the app setting '{0}' was not found. Please check the value of the setting.";
+        public const string MalformedConnectionStringDetails = "The connection string configured in app setting '{0}' is invalid (e.g.missing some required elements).";
+        public const string EventHubEntityNotFoundSummary = "The configured Event Hub '{0}' was not found.";
+        public const string EventHubEntityNotFoundDetails = "Refer to <a href='https://docs.microsoft.com/azure/azure-functions/functions-bindings-event-hubs-trigger#attributes' target'_blank'>relevant docs</a> and check the value of the attribute 'EventHubName' in your code.";
+        public const string ServiceBusEntityNotFoundSummary = "The configured Queue or Topic '{0}' was not found.";
+        public const string ServiceBusEntityNotFoundDetails = "Refer to <a href='https://docs.microsoft.com/azure/azure-functions/functions-bindings-service-bus-trigger#attributes' target'_blank'>relevant docs</a> and check the value of the attribute 'QueueName' or 'TopicName' in your code.";
+        public const string ManagedIdentityCredentialInvalidSummary = "The app setting '{0}" + Credential + "' is not valid.  To use identity based connections, set it's value to \"managedidentity\". See <a href='https://docs.microsoft.com/azure/azure-functions/functions-reference#common-properties-for-identity-based-connections' target='_blank'>relevant docs</a>. " + ManagedIdentityTutorial;
+        public const string KeyVaultReferenceResolutionFailedSummary = "The Azure Key Vault reference configured in app setting '{0}' could not be resolved. See < a href='https://docs.microsoft.com/azure/app-service/app-service-key-vault-references' target='_blank'>relevant docs</a> to correct the issue.";
```

Comment:
```suggestion
        public const string KeyVaultReferenceResolutionFailedSummary = "The Azure Key Vault reference configured in app setting '{0}' could not be resolved. See < a href='https://docs.microsoft.com/azure/app-service/app-service-key-vault-references' target='_blank'>relevant docs</a>.";
```

Reply:
Changed.  For summary,  shorter is always better :)